web version/app.py

```diff
@@ -36,20 +36,25 @@ def allowed_image_size(filesize):
     return True
 
 
+@app.route('/<string:page_name>')
```

Comment:
This will probably cause problems...
1. It means a user can now GET `/nonexist`. This method will try to get a non existent template and crash.
2. This will collide with any other path of the form: `/pattern`
I might be wrong, so I'll try to test it locally to see if works. But I'd appreciate any argument that says the contrary